src/plugin.js

```diff
@@ -107,9 +107,11 @@ const initPlugin = function(player, options) {
     // Disable timeouts in the background altogether according to the backgroundTimeout
     // option, or if the player is muted, as browsers may throttle javascript timers to
     // 1 minute in that case
+    const disableValues = [Infinity, -1];
+
     if ((document.visibilityState === 'hidden' &&
-          (options.backgroundTimeout === Infinity || player.muted())) ||
-        (document.visibilityState === 'visible' && options.timeout === Infinity)) {
+        (player.muted() || disableValues.includes(options.backgroundTimeout))) ||
+        (document.visibilityState === 'visible' && disableValues.includes(options.timeout))) {
```

Comment:
Wording in the readme suggests that we want feature parity for options.timeout and options.backgroundTimeout in terms of which values disable them. I can change the options.timeout condition back to what it was if we don't need it to accommodate `-1`, too.

Reply:
No, it makes sense to keep them matching.